README.md

```diff
@@ -61,11 +61,21 @@ once i had the data uploaded i entered code to find all of the different names w
 ```
 cou_edu.head(1)
 ```
-The line of code above allows me to see all of the collum headers with only 1 data result, with this information i am able to run the unique formula to establish what country names are in this dataset
-```
-cou_edu['country_name'].unique()
-``` 
-I can then take this and see the list of all the different countries that there are values for, this will enable me to see whether the list only consists of countries and not collectives like highlighted in the image above.
+The line of code above allows me to see all of the collum headers with only 1 data result, with this information i am able to run the unique formula to establish what country names are in this dataset. it was with this code that i was able to establish that there is 26 groups within this data that i need to get rid of.
+
+There is then 1 of 2 options, The first option is to import a list of all registed countries from the internet then merge this list with the dataset above, this will then remove all of the non-countries from the dataset above.
```

Comment:
```suggestion
Then I had two options.

The first option was to import a list of all registered countries from the internet then merge this list with the dataset above in such way that it would remove all of the non-countries from the dataset.
```